docs/Configuration.rst

```diff
@@ -2739,27 +2739,58 @@ type `` activity-type-key`` is enabled i.e it posts in the streams.
 To disable the activity type, simply set the parameter to ``false``.
 
 .. note:: In the ``exo.activity-type.activity-type-key.enabled``,
-		  activity-type-key could take many values depending on the activity
-		  type:
-
-			-  DEFAULT\_ACTIVITY
-			-  SPACE\_ACTIVITY
-			-  USER\_ACTIVITIES\_FOR\_SPACE
-			-  exosocial\\:relationship
-			-  LINK\_ACTIVITY
-			-  sharecontents\\:spaces
-			-  USER\_PROFILE\_ACTIVITY
-			-  DOC\_ACTIVITY
-			-  files\\:spaces
-			-  sharefiles\\:spaces
-			-  contents\\:spaces
-			-  cs-calendar\\:spaces
-			-  TaskAdded
-			-  ks-forum\\:spaces
-			-  ks-answer\\:spaces
-			-  ks-poll\\:spaces
-			-  ks-wiki\\:spaces
-			-  USER\_ACTIVITIES\_FOR\_RELATIONSHIP
+		  activity-type-key could take many values depending on the activity type.
+		  Learn more about different activity types below.
+		  
+``activity-type-key`` could take these values:
+
+			-  **DEFAULT_ACTIVITY**: For Activities posted by a user,  
+			   without documents or link attached.
+			-  **SPACE_ACTIVITY**: For activities about a space creation. 
+			   It contains the space's description and the number of 
+			   members.
+			-  **USER\_ACTIVITIES\_FOR\_SPACE**: An activity is posted when 
+			   a user creates an activity in a space.
+			-  **LINK\_ACTIVITY**: Activity with a link attachement.
+			-  sharecontents:spaces: An activity for contents sharing
```

Comment:
"sharecontents:spaces" should be emphasized.
And "Activity ..." instead of "An activity ...".